qa/integration/fixtures/certificates/certificate.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Ehh... I get what you are saying but a different key with same parameters
is unlikely to be the blame of a bug.

On Wednesday, September 14, 2016, Pier-Hugues Pellerin <
notifications@github.com> wrote:

> In qa/integration/fixtures/certificates/certificate.key
> <https://github.com/elastic/logstash/pull/5917#discussion_r78791590>:
>
> > @@ -0,0 +1,28 @@
> > +-----BEGIN PRIVATE KEY-----
>
> Good point, I think I wanted to make sure we have the same certificate so
> we can produce with it ?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/elastic/logstash/pull/5917/files/3a4572529c5d066cfbf8e6b7c053734c9568a67b#r78791590>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAIC6tCjEqX5ADLQqKEkhuMgxB2M9SHSks5qqCpTgaJpZM4J9AZ2>
> .
>